### PR TITLE
Add maunal testing for find in DG

### PR DESCRIPTION
**Pull request by w-yuchen**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -650,6 +650,62 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete property x` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
+<<<<<<< HEAD
+### Find commands
+
+1. Find
+
+    1. Test case: `find`<br>
+        Expected: Both list not updated, error message showing the two types of `find` commands is shown.
```
Comment (dvdweien) [inline docs/DeveloperGuide.md]: would "Both list are not updated" be better?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -650,6 +650,62 @@ testers are expected to do more *exploratory* testing.
...
+        Expected: `Woodlands Crescent` property is shown in the list, properties that do not contain the word `woodlands` in the name are not shown. 
+       
+    1. Test case: `find property pm/$700000 t/hdb`<br>
+        Expected: `Woodlands Crescent` property is shown in the list, properties with price less than $700,000 or are not HDB are not shown. 
+        
+    1. Test case: `find property n/`<br>
+        Expected: Property list not updated. Error details shown in the status message. 
+        
+    1. Test case: `find property pm/abc`<br>
+        Expected: Property list not updated. Error details shown in the status message.
```
Comment (dvdweien) [inline docs/DeveloperGuide.md]: would "Property list is not updated" be better?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -650,6 +650,62 @@ testers are expected to do more *exploratory* testing.
...
+        Expected: `Meet Emily` appointment is shown in the list, appointments that do not contain the word `emily` in the name are not shown. 
+        
+    1. Test case: `find appointment d/15-06-2021 t/1100`<br>
+        Expected: `Meet Emily` appointment is shown in the list, appointments whose date is not `Jun 15, 2021` and time is not `11:00AM` are not shown. 
+        
+    1. Test case: `find appointment n/ `<br>
+        Expected: Appointment list not updated. Error details shown in the status message. 
+        
+    1. Test case: `find appointment d/abc` <br>
+        Expected: Appointment list not updated. Error details shown in the status message.
```
Comment (dvdweien) [inline docs/DeveloperGuide.md]: would "Appointment list is not update." be better?

**Review by dvdweien — COMMENTED** [review]
Overall LGTM, just a couple of minor suggestions

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -650,6 +650,62 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete property x` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
+<<<<<<< HEAD
```
Comment (w2vgd) [inline docs/DeveloperGuide.md]: Do remove this

**Review by dvdweien — APPROVED** [review]
LGTM

**Pull request merged**